### PR TITLE
Typewriter

### PR DESCRIPTION
**Pull request by jillianchang**
This addresses #2

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -28,16 +28,16 @@ <h1><a href= "/" style="text-decoration: none; color: inherit;">Latin Scansion E
...
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/syllable.grm">syllable.grm</a></tt> syllabifies the phonetic transcriptions and subdivides into onset, nucleus, and coda.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/weight.grm">weight.grm</a></tt> codes syllables as heavy or light.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/foot.grm">foot.grm</a></tt> groups sequences of heavy and light syllables into dactyls, spondees, and trochees.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/hexameter.grm">hexameter.grm</a></tt> filters foot sequences to only those permitted in a hexameter verse.</li>
+							<li><span class="tt"><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/normalize.grm">normalize.grm</a></span> removes punctuation and case-folds.</li>
```
Comment (jillianchang) [inline templates/base.html:34]: The tt tag is depreciated from HTML 5 and no longer recommended. I used CSS instead to create the same look.

**Comment by jillianchang** [comment]
Previous version
Screen Shot 2022-03-14 at 5 11 26 PM

New version
Screen Shot 2022-03-14 at 5 11 13 PM

A note that I removed the underlines from the links because I felt that the underlines make it hard to discern the typewriter style. Now only upon hover do the links underline.

File: templates/base.html (hunk 1 above)
Comment (kylebgorman) [reply templates/base.html:34]: Oh I had no idea. Thanks for sussing that out. (In this case I generated the HTML using a Markdown converter.)

**Review by kylebgorman — COMMENTED** [review]
LGTM

**Comment by kylebgorman** [comment]
> A note that I removed the underlines from the links because I felt that the underlines make it hard to discern the typewriter style. Now only upon hover do the links underline.

Thanks, that sounds great.

**Pull request merged**